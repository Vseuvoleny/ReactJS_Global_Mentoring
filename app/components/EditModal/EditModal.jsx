import React from "react";
import PropTypes from "prop-types";
import MovieForm from "../MovieForm/MovieForm";
import "./EditModal.scss";

const EditModal = () => {
  return (
    <section className="edit-modal">
      <span className="edit-modal__title">edit movie</span>
      <MovieForm />
    </section>
  );
};

EditModal.propTypes = {};

export default EditModal;
