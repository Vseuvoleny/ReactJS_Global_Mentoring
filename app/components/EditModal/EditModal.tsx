import React from "react";
import MovieForm from "../MovieForm/MovieForm";
import classes from "./EditModal.module.scss";

const EditModal = () => {
  return (
    <section className={classes.edit_modal}>
      <span className={classes.edit_modal__title}>edit movie</span>
      <MovieForm />
    </section>
  );
};

export default EditModal;
