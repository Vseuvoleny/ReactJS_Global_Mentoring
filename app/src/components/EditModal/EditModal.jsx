import React from "react";
import PropTypes from "prop-types";
import Button from "../../Elements/Button/Button";

const EditModal = () => {
  return (
    <section className="edit-modal">
      <span>EDIT MOVIE</span>
      <form></form>
    </section>
  );
};

EditModal.propTypes = {};

export default EditModal;
