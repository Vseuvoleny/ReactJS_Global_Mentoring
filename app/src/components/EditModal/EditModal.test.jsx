import React from "react";
import { render } from "@testing-library/react";
import EditModal from "./EditModal";

jest.mock("../MovieForm/MovieForm", () => () => <div>Movie Form</div>);

describe("EditModal component", () => {
  test("should return MovieForm", () => {
    const { container, getByText } = render(<EditModal />);
    expect(container.textContent).toMatch("Movie Form");
    expect(getByText("edit movie")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
