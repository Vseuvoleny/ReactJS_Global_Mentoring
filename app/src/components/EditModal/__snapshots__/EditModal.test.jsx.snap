// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditModal component should return MovieForm 1`] = `
<div>
  <section
    class="edit-modal"
  >
    <span
      class="edit-modal__title"
    >
      edit movie
    </span>
    <div>
      Movie Form
    </div>
  </section>
</div>
`;
